src/style/style.css

```diff
@@ -0,0 +1,4 @@
+body {
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif !important;
+    user-select: none;
```

Comment:
user-select has its own class within tailwindcss called `select-none`. I'd prefer if it was used.